code/game/objects/items/bodybag.dm

```diff
@@ -14,8 +14,8 @@
 
 
 /obj/item/storage/box/bodybags
-	name = "body bags"
-	desc = "This box contains body bags."
+	name = "\improper box of body bags"
+	desc = "This box consists of body bags, each with a written label, this particular box includes wirecutters to remove labels."
```

Comment:
WAY too long

Reply:
body bags arent used as much as you'd expect- always examined.